CHANGELOG.md

```diff
@@ -36,6 +36,7 @@ v1.10.0-rc.0
 - (_Experimental_) Add an `array.group_by` stdlib function to group items in an array by a key. (@wildum)
 - Add the `otelcol.exporter.faro` exporter to export traces and logs to Faro endpoint. (@mar4uk)
 - Add the `otelcol.receiver.faro` receiver to receive traces and logs from the Grafana Faro Web SDK. (@mar4uk)
+- Add the `otelcol.receiver.fluentforward` receiver to receive logs via Fluent Forward Protocol. (@rucciva)
```

Comment:
Could you please move this to the "main" section of the changelog? It is currently in the 1.10.0 RC section. Apologies for this, I'll merge the PR right after this is fixed and after it's rebased.

Reply:
got it, no worries